### ISSUE TITLE
FastBreak anticheat bugs

### ISSUE THREAD

**Issue by mathiascode** [cuberite/cuberite#3506]
The FastBreak anticheat currently has a few issues. Blocks such as ice, rails and pressure plates cannot be dug with a pickaxe, instead the "FastBreak?" message appears in chat. It seems like other Cuberite users have had issues with FastBreak as well (https://forum.cuberite.org/thread-2660.html), so I suggest that we disable the anticheat temporarily until the issue is fixed.

**Comment by mohe2015** [comment]
Yes, we should do that. If I find time I'll try to fix the issue.
